docs/regression_test_api.rst

```diff
@@ -87,18 +88,20 @@ In essence, these builtins exert control over the test creation, and they allow
   One of the most powerful features about these built-in functions is that they store their input information at the class level.
   However, a parameter may only be accessed from the class instance and accessing it directly from the class body is disallowed.
   With this approach, extending or specializing an existing parametrized regression test becomes straightforward, since the test attribute additions and modifications made through built-in functions in the parent class are automatically inherited by the child test.
```

Comment:
```suggestion
  With this approach, extending or specializing an existing parameterized regression test becomes straightforward, since the test attribute additions and modifications made through built-in functions in the parent class are automatically inherited by the child test.
```

Reply:
I've made sure `parametrized` is fully exterminated from the docs :)